### ISSUE TITLE
(A11y severity 3) Add minimal screen reader support to collabdocs

### ISSUE THREAD

**Issue by sathomas** [oaeproject/3akai-ux#3953]
Might be possible to fix this easily and send PR to etherpad:

* Screen reader users are unable to edit or add text to this frame unless they manually enable application mode. Since most text entry forms don’t require users to manually change modes, this can be unintuitive and should be addressed if possible.

* Currently there is no form of heading or navigation structure within the editable document. Adding headings that can be navigated to within the document would greatly assist screen reader users.

**Comment by sathomas** [comment]
Fixed with in Etherpad 1.5.6
